src/test/java/com/exactpro/th2/mstore/TestErrorCollector.java

```diff
@@ -143,7 +142,6 @@ void testLogAndCollect() throws IOException {
         assertTrue(body.matches("\\[\\{\"errors\":\\{\"A\":\\{\"firstDate\":\"\\d+-\\d+-\\d+T\\d+:\\d+:\\d+.\\d+\",\"quantity\":1}}}]"), () -> "body: " + body);
 
         taskCaptor.getValue().run();
-        verifyNoMoreInteractions(eventRouter);
```

Comment:
Could you let me know why you've removed this check? Do we expect some events to be sent after the second call of `run` method?

Reply:
This class contains afterEach where all mock objects are checked